meerkat_frontend/src/js/technical/overview.js

```diff
@@ -200,10 +200,7 @@ function prep_row_draw_Last3(contentsObj, parentId, locID) {
                 return new Date(b.date) - new Date(a.date);
             });
 
-            //Take the last 3 values so i need to reverse the array ..
-            arrValue.reverse();
-            arrDate.reverse();
-
+            
```

Comment:
This creates a conflict with the development branch that needs resolving.  It is a really quick and easy one.  Just click on resolve conflicts" in the pull request.